ocs_ci/helpers/network_policy_helpers.py

```
@@ -0,0 +1,386 @@
+            if "yes" not in str(result).lower():
+                failed_verbs.append(verb)
+        except CommandFailed:
+            failed_verbs.append(verb)
+
+    assert not failed_verbs, (
+        f"SA {sa_name} cannot {failed_verbs} NetworkPolicies "
+        f"in {namespace}"
+    )
+    logger.info(
+        f"SA {sa_name} has create/delete/update permissions for "
+        f"NetworkPolicies in {namespace}"
```

Ruff (0.16.0)

[warning] Logging statement uses f-string

(G004)